baybe/utils/basic.py

```diff
@@ -54,6 +54,42 @@ def get_subclasses(cls: _C, recursive: bool = True, abstract: bool = False) -> l
     return subclasses
 
 
+def get_parent_classes(
+    cls: type,
+    recursive: bool = True,
+    abstract: bool = False,
+    include_class: bool = True,
```

Comment:
I think it makes more sense to have this default to `False` actually, otherwise could be confusing. What does @Scienfitz think?

Reply:
agree